port/unix/omrsysinfo.c

```diff
@@ -4476,11 +4477,11 @@ omrsysinfo_get_cgroup_subsystem_list(struct OMRPortLibrary *portLibrary)
  * Returns TRUE if running inside a container and FALSE if not or if an error occurs
  */
 BOOLEAN
-omrsysinfo_is_running_in_container(struct OMRPortLibrary *portLibrary, int32_t *errorCode)
+omrsysinfo_is_running_in_container(struct OMRPortLibrary *portLibrary)
 {
 	BOOLEAN inContainer = FALSE;
 #if defined(LINUX) && !defined(OMRZTPF)
-	*errorCode = isRunningInContainer(portLibrary, &inContainer);
+	inContainer = PPG_isRunningInContainer;
```

Comment:
This function can just return `PPG_isRunningInContainer`. I don't think there is any need for `inContainer` here.

Reply:
Would it make sense to have the global on all platforms and always just return the global?  Platforms that do not support containers can just always set the global to false.  This way in the future if other platforms do support containers there would be less code to change.

Reply:
@charliegracie That sounds good to me, Will be making the changes. Thanks !